common/pref_names.h

```diff
@@ -21,9 +21,9 @@ extern const char kFirstCheckMade[];
 extern const char kThresholdCheckMade[];
 extern const char kThresholdQuery[];
 extern const char kWeekOfInstallation[];
-extern const char kAdBlockCheckedDefaultRegion[];
-extern const char kAdBlockCustomFilters[];
-extern const char kAdBlockRegionalFilters[];
+// extern const char kAdBlockCheckedDefaultRegion[];
+// extern const char kAdBlockCustomFilters[];
+// extern const char kAdBlockRegionalFilters[];
```

Comment:
Looks like you can remove this commented lines, these are now in components/brave_shields/common/pref_names.h

Reply:
oops, good catch. Removing